15-synthesis.Rmd

```diff
@@ -55,10 +55,11 @@ The range of options grows further when other R packages are considered: you cou
 We recommend using **sf** and the other packages showcased in this book, for reasons outlined in Chapter \@ref(spatial-class), but it's worth being aware of alternatives and being able to justify your choice of software.
 
 We deliberately covered both **tidyverse** and base R approaches to attribute data operations.
-Chapter \@ref(attr) showed how both `nz["Name"]` and `nz %>% select(Name)` can be used to achieve the same result, for example.
+Chapter \@ref(attr) showed how both `nz[, "Name"]` and `nz %>% select(Name)` can be used to achieve the same result, for example.
```

Comment:
@jannes-m The previous version was probably better. 
- On sf object, both `nz["Name]` and `nz[, "Name"]` give the same result
- However on data.frame objects `nz_df["Name"]` is the same as `nz_df %>% select(Name)`
- `nz_df[, "Name"]` is more like `nz_df %>% pull(Name)`

Reply:
Robin, while on the bike today,what do you think about replacing "Geocomputation for social benefit" by "Advocating open source software"? This is broader, we all agree, and you can still say that using an open source approach, social benefits are more likely to happen.

Reply:
@jannes-m `nz["Name"]` and `nz[, "Name"]` give the same output - an sf object with two columns. This is due to "stickiness" of a geometry column.